pkg/reconciler/taskrun/controller.go

```diff
@@ -70,7 +70,7 @@ func NewController(ctx context.Context, cmw configmap.Watcher) *controller.Impl
 		}
 	})
 
-	taskRunInformer.Informer().AddEventHandler(controller.HandleAll(impl.Enqueue))
+	taskRunInformer.Informer().AddEventHandler(controller.HandleAll(impl.Enqueue)) //nolint:errcheck
```

Comment:
Instead of suppressing the errcheck, should we log instead?

Reply:
Done!